File: Selenium/Project/Activity1.java
package SeleniumLMS;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.asserts.Assertion;

public class Activity1 {

    public static void main(String[]args){
        WebDriverManager.firefoxdriver().setup();

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"NULL");

        WebDriver driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/lms");



        String Title = driver.getTitle();
        System.out.println(Title);



        if(Title.contains("Alchemy LMS – An LMS Application"))
        {
            Assert.assertEquals("Alchemy LMS – An LMS Application",Title);
            driver.close();
        }
        else
        {
            System.out.println("Landed on wrong page");

        }


    }

}

File: Python/Project/Activity1.py
import options as options
from selenium import webdriver

driver = webdriver.Chrome()


def openURL():
    options = webdriver.ChromeOptions()
    options.add_experimental_option("detach", True)
    # driver = webdriver.Chrome(options=options)
    driver.get('https://alchemy.hguy.co/lms')


def verifyPageTitle():
    get_title = driver.title
    print('Page title is: ', get_title)
    assert get_title == 'Alchemy LMS – An LMS Application'
    print('Assertion completed successfully')

def closeBrowser():
    driver.quit()

# Open the URL
openURL()

# Verify the Page Title
verifyPageTitle()

# Close the browser
closeBrowser()